viz/cmd/main_test.go

```diff
@@ -0,0 +1,87 @@
+package cmd
```

Comment:
This whole file is pretty much the same as `cli/cmd/main_test.go`, right? Maybe it would make sense to move these functions into `pkg/` and just have the `main_test.go`s call into that?